### DISCUSSION TITLE
How is the 'parent' prop on the Router component supposed to be used?

### DISCUSSION THREAD

**Discussion by astanb** [molefrog/wouter#303] [Q&A] [answered]
There seems to be no documentation on this prop.

**Comment by molefrog** [comment, marked as answer]
Thanks, sorry for my long reply. I've added it to the [README](https://github.com/molefrog/wouter#are-relative-routes-and-links-supported).

Btw, we've just published the [v3.0.0-rc.1](https://github.com/molefrog/wouter/releases/tag/v3.0.0-rc.1) that removes `parent` prop and makes it easier to nest routes. `parent` prop is no longer needed because a Router will always inherit from its parent.
